examples/205-automl.ipynb

```diff
@@ -25,6 +25,7 @@
     "- [General AutoML](#chapter_2)\n",
```

Comment:
<p>Probably better to write:</p><p> </p><p>We can define our own set of pipelines for the search.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>